blog/scss/_typography.scss

```diff
@@ -0,0 +1,35 @@
+body {
+  font-family: $primaryFontFamily;
+  overflow-x: hidden;
+
+  * {
+    padding: 0;
+    margin: 0;
+    font: revert;
+  }
+
+  a {
+    text-decoration: none;
+    color: $black;
+  }
+
+  a:hover {
+    transition: 0.5s;
+    color: $lightGray;
+  }
+
+  li {
+    list-style-type: none;
+  }
+
+  .container {
+    width: 70%;
+  }
+
+  .separator {
+    height: 1px;
+    background-color: $silverMist;
+    margin: nth($spaces, 4) 0;
+    width: 100%;
+  }
+}
```

Comment:
I think some of the styles are more related to base styles, so I think creating a new file like "_base.scss" in blog folder would be cleaner :)